hardware/1Wire.cpp

```diff
@@ -32,51 +32,38 @@ C1Wire::C1Wire(const int ID, const int sensorThreadPeriod, const int switchThrea
 {
 	m_HwdID = ID;
 
-	// Defaults for pre-existing 1wire instances
-	if (0 == m_sensorThreadPeriod)
```

Comment:
Maybe check if it's not 0, and < 5 * 60 * 1000 then 5 * 60 * 1000 ? else people will enter 1
Or we should change it in seconds/minutes (also in the web gui) ?